src/lib/utils/unescape.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This implementation is taken from Lodash implementation.
+ * See: https://github.com/lodash/lodash/blob/4.17.11-npm/unescape.js
+ */
+
+// Used to map HTML entities to characters.
+const htmlEscapes = {
+  '&amp;': '&',
+  '&lt;': '<',
+  '&gt;': '>',
+  '&quot;': '"',
+  '&#39;': "'",
+};
+
+// Used to match HTML entities and HTML characters.
+const regexEscapedHtml = /&(amp|quot|lt|gt|#39);/g;
```

Comment:
Can this be constructed from `htmlEscapes`?

Reply:
I'm not sure on how to do it without adding a loop to get all the keys